.travis.yml

```diff
@@ -32,5 +32,17 @@ script:
 
 after_success:
   # push coverage results to Codecov
-  - julia --project -e 'import Pkg; Pkg.add("Documenter"); include("docs/make.jl")'
-  - julia --project -e 'import Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())'
+  - julia -e 'using Pkg; cd(Pkg.dir("PowerSystems")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
```

Comment:
We Should also remove this line since we don't use coveralls.

Reply:
Resolved in 2e373c0.